setup.sh

```diff
@@ -0,0 +1,382 @@
+  while [[ "$launchTheStack" != [yYnN] ]]
+    do
+      promptBold "[❓] Do you want to start Kuzzle now? (y/N) "
+      read launchTheStack trash
+      case "$launchTheStack" in
+        [yY])
+          echo
+          writeBold "Starting Kuzzle..."
+          $DOCKER_COMPOSE_BIN -f $composerYMLPath up -d
+          echo
+          write "$GREEN" "[✔] Kuzzle is up and running!"
+          ;;
+        [nN] | '')
+          echo
+          writeBold "$BLUE" "Ok."
+          launchTheStack=n
+          ;;
+        *)
+          echo
+          writeBold "$RED" "[✖] I did not understand your answer."
+          ;;
+      esac
+    done
+  echo
+  writeBold "Where do we go from here?"
+  shortHelp
+}
+
+shortHelp() {
+  echo
+  writeBold "* You can open this short help by calling ./setup.sh --help"
+  writeBold "* You can start Kuzzle by typing:"
+  write "  docker-compose -f $composerYMLPath up -d"
+  writeBold "* You can see the logs of the Kuzzle stack by typing:"
+  write "  docker-compose -f $composerYMLPath logs -f"
+  writeBold "* You can check if everything is working by typing:"
+  write "  curl -XGET http://localhost:7512/"
+  writeBold "* You can stop Kuzzle by typing:"
+  write "  docker-compose -f $composerYMLPath stop"
+  writeBold "* You can restart Kuzzle by typing:"
+  write "  docker-compose -f $composerYMLPath restart"
+  writeBold "* You can read the docs at http://docs.kuzzle.io/"
+  echo
+}
+
+# Main execution routine
+# ===========================
+
+if [ "$1" == "--help" ]; then
+  echo
+  writeBold "# Kuzzle short help"
+  writeBold "  ================="
+  shortHelp
+  exit 0
+fi
+
+echo
+writeBold "# Kuzzle Setup"
+writeBold "  ============"
+echo
+writeBold "This script will help you launch Kuzzle and install"
+writeBold "all the necessary dependencies."
+echo
+write     "* You can refer to http://docs.kuzzle.io/ if you need better"
+write     "  understanding of the installation process."
+write     "* Feel free to join us on Gitter at https://gitter.im/kuzzleio/kuzzle"
+write     "  to get help in real time."
+
+checkRoot
+
+CHECK_ARCH=$(uname -a | grep x86_64)
+if [ -z ${CHECK_ARCH} ]; then
+  echo
+  writeBold "$RED" "[✖] Kuzzle runs on x86_64 architectures, which does not seem"
+  writeBold "$RED" "    to be the architecture of your system."
+  write            "    Sorry, you cannot launch Kuzzle on this machine."
+  echo
+  exit 4
+fi
+
+# TODO check available memory
+findOSType
+isOSSupported
+
+if ! commandExists docker; then
+  installDocker
+fi
+
+CHECK_DOCKER_RUN=$(docker run hello-world &> /dev/null)
+if ! ${CHECK_DOCKER_RUN} ; then
+  echo
+  writeBold "$RED" "[✖] Docker does not seem to be running on your system."
+  writeBold "$RED" "    Please start the Docker daemon and re-run this script"
+  write            "    More information at https://docs.docker.com/engine/admin/"
+  echo
+  exit 5
+fi
+
+if ! commandExists $DOCKER_COMPOSE_BIN; then
```

Comment:
`if ! commandExists docker-compose `

Because `docker-compose` may not be installed in the default directory set at the beginning of this script.